### DISCUSSION TITLE
Add a option to turn the task bar search bar on and off

### DISCUSSION THREAD

**Discussion by Cherrrrryyy** [valinet/ExplorerPatcher#1667] [Ideas]
I don't know if it's just an issue for me but I can't get the search bar on my task bar, if this is simply not able to be implemented then that's my bad it is probably written somewhere, if it isn't though I would very much appreciate this feature and think other people would too :3

**Comment by mikal-s** [comment]
Hi Can set wither search icon or search bar, or both show, in windows setting>Personalisation>Taskbar
Or can go there from Patcher Properties window, right click taskbar, choose properties then click taskbar in property window, use > More taskbar options.. will take you to same place..

**Reply by Cherrrrryyy** [reply]
I am perhaps experiencing a bug then because I have it set to having the search bar on and it is not appearing

**Reply by mikal-s** [reply]
In ExplorerPatcher you need to set taskbar to windows 11 which is default, if set for windows 10 you can only get the mag-glass icon.
that seems to be how works on my computers.

**Reply by Fa310tx** [reply]
I can only get the icon, as well, no bar.
I would prefer that option, as it's most like default Windows 10.
JJ

**Comment by XuanChen1234** [comment]
Any more project?

**Comment by XuanChen1234** [comment]
Any more planned?

**Comment by pyrates999** [comment]
You can contribute these features to explorer patcher if you like.

**Comment by milestprower92** [comment]
No longer possible in 22H2 and 23H2 because Microsoft has removed the code of the old search box from the old taskbar and there are no plans to have it added back due to the fact that the search icon does the same thing as the search box.

**Comment by GitEin11** [comment]
i hope they return it  because I use music player
by zhongyang219
https://github.com/zhongyang219/MusicPlayer2

that utilize it as lyrics viewer
![Cortana_lyric](https://github.com/valinet/ExplorerPatcher/assets/84138811/36753977-667d-4bb1-8986-47d230cce0d0)
